src/libraries/Microsoft.PowerFx.Interpreter/Environment/SymbolExtensions.cs

```diff
@@ -41,9 +41,10 @@ public static void SetCulture(this RuntimeConfig symbols, CultureInfo culture)
         /// <param name="symbols">SymbolValues where to set the CultureInfo.</param>
         /// <param name="clock">service to provide current time.</param>
         /// <exception cref="ArgumentNullException">When clock is null.</exception>
-        public static void SetClock(this RuntimeConfig symbols, IClockService clock)
+        public static RuntimeConfig SetClock(this RuntimeConfig symbols, IClockService clock)
```

Comment:
>RuntimeConfig [](http://example.com/codeflow?start=22&length=13)

let's not - 
the other similar APIs in this file don't do this; and this is also an API breaking change.